src/reducers/data.js

```diff
@@ -48,11 +56,11 @@ const stats = (state = {}, action) => {
   switch (action.type) {
     case GET_RIGHT:
     case GET_WRONG:
-      var f = {
-        ...state,
-        [action.card.hint]: byHint(state[action.card.hint], action)
-      };
-      return f;
+      return Object.assign({}, state, {[action.card.hint]: byHint(state[action.card.hint], action)});
```

Comment:
@frankiefu i think this may still be wrong? i dont understand the sorcery you did to translate the other one tho